src/wizard/javascript/react.md

```diff
@@ -15,39 +20,44 @@ yarn add @sentry/react @sentry/tracing
 npm install --save @sentry/react @sentry/tracing
 ```
 
+## Configure
+Initialise Sentry as early as possible in your application's lifecycle.
```

Comment:
Can we combine this sentence with the next one, it seems ambiguous.

Reply:
@HazAT let me know if I've misunderstood what you meant

From the usability tests I've witnessed, engineers tend to copy these chunks of code without thinking. So whilst I would understand the drive to simply 3 steps into 2, I assume it would likely lead to more human error.